src/dataviews/dataview-model-base.js

```diff
@@ -0,0 +1,121 @@
+var Model = require('../core/model');
+
+/**
+ * Default dataview model
+ */
+module.exports = Model.extend({
+  defaults: {
+    url: '',
+    data: [],
+    columns: [],
+    sync: true,
+    bbox: true,
+    disabled: false
+  },
+
+  url: function () {
+    return this.get('url') + '?bbox=' + this.get('boundingBox');
+  },
+
+  initialize: function (attrs, opts) {
+    attrs = attrs || {};
+    opts = opts || {};
+
+    if (!attrs.id) {
+      this.set('id', attrs.type + '-' + this.cid);
+    }
+
+    this.layer = opts.layer;
+
+    // filter is optional, so have to guard before using it
+    this.filter = opts.filter;
+    if (this.filter) {
+      this.filter.set('dataviewId', this.id);
+    }
+
+    this._initBinds();
+  },
+
+  _initBinds: function () {
+    this.once('change:url', function () {
+      var self = this;
+      this._fetch(function () {
+        self._onChangeBinds();
+      });
+    }, this);
+
+    // Retrigger an event when the filter changes
+    if (this.filter) {
+      this.filter.bind('change', this._onFilterChanged, this);
+    }
+  },
+
+  _onChangeBinds: function () {
+    this.bind('change:url', function () {
+      if (this.get('sync') && !this.isDisabled()) {
```

Comment:
As mentioned offline, IMO `sync` and `bbox` attributes shouldn't be part of dataview models. Widget models are the ones that determine if the widget is synced to changes on other widgets (`sync`) and changes on the bounding box (`bbox`). I think that dataviews should trigger and event every time the "datasource" changes, and then the widgets should decide what to do based on their configuration (e.g. fetch the data again and refresh themselves if `sync` is enabled).